### DISCUSSION TITLE
Any Success or Demo video of the same

### DISCUSSION THREAD

**Discussion by parmarsumit** [aws-solutions/workload-discovery-on-aws#451] [Q&A]
I have tried multiple times this CFN stack doesnt work .
Paramters :
AdminUserEmailAddress	abc@xyz.com	-
AlreadyHaveConfigSetup	Yes	-
AthenaWorkgroup	primary	-
CreateNeptuneReplica	No	-
CreateOpensearchServiceRole	Yes	-
NeptuneInstanceClass	db.r5.large	-
OpensearchInstanceType	m6g.large.search	-
OpensearchMultiAz	No	-
OptOutOfSendingAnonymousUsageMetrics	No

Error: the following resource(s) failed to create: [VpcStack, AthenaGlueCrawlerStack, WebUiStack]. Rollback requested by user.
AthenaGlueCrawlerStack	CREATE_FAILED Resource creation cancelled
WebUiStack	CREATE_FAILED Resource creation cancelled
VpcStack	CREATE_FAILED.   Embedded stack arn:aws:cloudformation:ap-southeast-2:357085863033:stack/wdiscover-VpcStack-14BA7KRNJ2NOJ/6739e0c0-50f5-11ee-9455-0aac01a06828 was not successfully created: The following resource(s) failed to create: [ElasticIP1].

**Comment by ctd** [comment]
The error message indicates an error creating an Elastic IP resource. As an immediate troubleshooting step, can you check you have not reached the [Elastic IP limit](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html#using-instance-addressing-limit) in the AWS account you're trying to deploy to?

Otherwise, if you look at the VpcStack resource in CloudFormation you may be able to find more details for the error.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
